cwl/src/test/scala/wdl4s/cwl/ExportCwlSamplesSpec.scala

```diff
@@ -0,0 +1,64 @@
+package wdl4s.cwl
+
+import io.circe.generic.auto._
+import org.scalatest.{FlatSpec, Matchers}
+import io.circe.shapes._ // required, even though IntelliJ thinks it is unused
+import io.circe.syntax._
+import io.circe.refined._
+import io.circe.literal._
+import wdl4s.cwl.CwlEncoders._
+import shapeless.Coproduct
+
+/**
+  * wdl4s
+  * Created by oruebenacker on 03.07.17.
+  */
+class ExportCwlSamplesSpec extends FlatSpec with Matchers {
+
+  it should "export 1st tool" in {
+    val tool =
+      CommandLineTool(
+        inputs = Coproduct[CommandLineTool.Inputs](Map("message" -> CommandInputParameter(
+          id = None,
+          label = None,
+          secondaryFiles = None,
+          format = None,
+          streamable = None,
+          doc = None,
+          inputBinding = Some(CommandLineBinding(
+            loadContents = None,
+            position = Some(1),
```

Comment:
For literals it doesn't really matter, but we try to enforce always preferring `Option` over `Some` both for muscle memory purposes and that way to ensure that nothing changes down the road such that a non-literal could find its way in there